cookbooks/bcpc-hadoop/templates/default/zk_zookeeper-env.sh.erb

```diff
@@ -12,7 +12,7 @@ export ZOO_PID_DIR=/var/run/zookeeper
 export ZOOPIDFILE=$ZOO_PID_DIR/zookeeper-server
 export ZOOCFGDIR=<%= node[:bcpc][:hadoop][:zookeeper][:conf_dir] %>
 export ZOO_DATADIR_AUTOCREATE_DISABLE=1
-SERVER_JVMFLAGS="$SERVER_JVMFLAGS -Djute.maxbuffer=<%= node[:bcpc][:hadoop][:jute][:maxbuffer]  %>"
+SERVER_JVMFLAGS="$SERVER_JVMFLAGS -Djute.maxbuffer=<%= node[:bcpc][:hadoop][:jute][:maxbuffer]  %> <%= node[:bcpc][:jolokia][:jvm_args] %>"
```

Comment:
FC001 - Use strings in preference to symbols to access node attributes

(sorry)